elephantagenda/apps.py

```diff
@@ -0,0 +1,7 @@
+
+from django.apps import AppConfig
+
+
+class AgendaConfig(AppConfig):
+    name = 'elephantagenda'
+    verbose_name = "Agenda"
```

Comment:
Please make `verbose_name` translatable (using `ugettext_lazy`) and keep it lowercase.